src/index.js

```diff
@@ -1,13 +1,74 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
+class TodoData {
+  constructor() {
+    if (window.localStorage.getItem('tasksData') !== undefined) {
+      this.tasksData = JSON.parse(window.localStorage.getItem('tasksData'));
+    } else {
+      this.tasksData = [];
+    }
   }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+
+  addTaskEntry(task) {
```

Comment:
- [ ] Dear @EnixWord, as part of this milestone, we need to create a new javascript file for the new functionalities. Please create a new file and store the functionalities in a new `.js` file.